_posts/2022-01-20-saying-no.md

```diff
@@ -0,0 +1,43 @@
+
+### Why say "no"
+
+I've been consuming a lot of Brené Brown's [podcasts](https://brenebrown.com/podcasts/) and read [one of her books](https://brenebrown.com/book/atlas-of-the-heart/) recently, and they've got me thinking about what's at the root of being able to say "no" regularly and well.
+
+[![BRAVING Trust]({{ '/images/a/braving.jpg' | absolute_url }})](https://www.linkedin.com/pulse/braving-trust-brené-brown/)
+
+Brown uses the [BRAVING acronym](https://www.linkedin.com/pulse/braving-trust-brené-brown/) to refer to the elements of Trust. Out of these, the ones that jump out to me for saying no are boundaries, reliability and integrity. To meaningfully trust others on these axes, you cannot be someone who says yes to every request. In doing so, you maintain no boundaries, professionally or personally, and will not be a reliable coworker.
+
+To the software engineers reading this, I'm sure we've all worked with some APIs that have been great and some that have been terrible. An API with inconsistent and unpredictable behaviour with the same inputs is terrible. A poorly documented API is terrible. An API that attempts to accommodate your request and consistently times out rather than failing immediately is terrible.
+
+Humans are like APIs in this regard. A bad API for your coworkers is to be someone who gives radically different estimates for the same work based on whether you want to impress the person or not. A bad API for your coworkers is when you state you're on holiday or sick but still appear in conversations and meetings anyway. A bad API for your coworkers is when you tell them a 19:00 meeting is fine, but you're hangry and irritable because you've missed dinner and your children's bedtime.
+
+A previous performance of mine from [a great manager](https://github.com/derekprior) started with the sentence "Mike is very strict with his boundaries". "Oh no", I thought to myself, "I'm going to get criticised for always prioritising being there for my children's dinner and bedtimes four nights of the week". Instead, he went on to say that this made me easier to work with, set an excellent example for other members of the team to not overwork and set expectations for new parents on the team. Ironically, in this case, my being a little more selfish with my time and strict with my boundaries produced an outcome that benefitted the entire team.
+
+Having these boundaries clear and upfront is more complex but crucial for unexpected tasks. If you already feel overwhelmed or have too much to do when asked to do something new: you should say "no". If when asked for an estimate, [you can tell there's a bunch of complexity that was not considered by those who requested the estimate]({{ '/2021/11/29/hidden-complexities-of-software-estimation/' | absolute_url }}) and know that your realistic one will disappoint them: you should give them the realistic estimate.
```

Comment:
- It seems like a comma might be needed here
- Might help clean up what felt like a long sentence
```suggestion
Having these boundaries clear and upfront is more complex, but crucial for unexpected tasks. If you already feel overwhelmed or have too much to do when asked to do something new: you should say "no". If when asked for an estimate, [you can tell there's a bunch of complexity that was not considered by the requester(s)]({{ '/2021/11/29/hidden-complexities-of-software-estimation/' | absolute_url }}) and know that your realistic one will disappoint them: you should give them the realistic estimate.
```